packages/cli/test/integration/evaluation/test-definitions.api.test.ts

```diff
@@ -0,0 +1,296 @@
+import { Container } from 'typedi';
+
+import type { AnnotationTagEntity } from '@/databases/entities/annotation-tag-entity.ee';
+import type { User } from '@/databases/entities/user';
+import type { WorkflowEntity } from '@/databases/entities/workflow-entity';
+import { TestDefinitionRepository } from '@/databases/repositories/test-definition.repository.ee';
+import { createAnnotationTags } from '@test-integration/db/executions';
+
+import { createUserShell } from './../shared/db/users';
+import { createWorkflow } from './../shared/db/workflows';
+import * as testDb from './../shared/test-db';
+import type { SuperAgentTest } from './../shared/types';
+import * as utils from './../shared/utils/';
+
+let authOwnerAgent: SuperAgentTest;
+let workflowUnderTest: WorkflowEntity;
+let evaluationWorkflow: WorkflowEntity;
+let otherWorkflow: WorkflowEntity;
+let ownerShell: User;
+let annotationTag: AnnotationTagEntity;
+const testServer = utils.setupTestServer({ endpointGroups: ['evaluation'] });
+
+beforeAll(async () => {
+	ownerShell = await createUserShell('global:owner');
+	authOwnerAgent = testServer.authAgentFor(ownerShell);
+});
+
+beforeEach(async () => {
+	await testDb.truncate(['TestDefinition', 'Workflow', 'AnnotationTag']);
+
+	workflowUnderTest = await createWorkflow({ name: 'workflow-under-test' }, ownerShell);
+	evaluationWorkflow = await createWorkflow({ name: 'evaluation-workflow' }, ownerShell);
+	otherWorkflow = await createWorkflow({ name: 'other-workflow' });
+	annotationTag = (await createAnnotationTags(['test-tag']))[0];
+});
+
+describe('GET /evaluation/test-definitions', () => {
+	test('should retrieve empty test definitions list', async () => {
+		const resp = await authOwnerAgent.get('/evaluation/test-definitions');
+		expect(resp.statusCode).toBe(200);
+		expect(resp.body.data.count).toBe(0);
+		expect(resp.body.data.testDefinitions).toHaveLength(0);
+	});
+
+	test('should retrieve test definitions list', async () => {
+		const newTest = Container.get(TestDefinitionRepository).create({
+			name: 'test',
+			workflow: { id: workflowUnderTest.id },
+		});
+		await Container.get(TestDefinitionRepository).save(newTest);
+
+		const resp = await authOwnerAgent.get('/evaluation/test-definitions');
+
+		expect(resp.statusCode).toBe(200);
+		expect(resp.body.data.count).toBe(1);
+		expect(resp.body.data.testDefinitions).toHaveLength(1);
+		expect(resp.body.data.testDefinitions[0].name).toBe('test');
+		expect(resp.body.data.testDefinitions[0].workflowId).toBe(workflowUnderTest.id);
+		expect(resp.body.data.testDefinitions[0].evaluationWorkflowId).toBe(null);
```

Comment:
nit: This is a matter of preference, but personally I would find this be more readable:

```ts
expect(resp.body.data).toEqual({
  count: 1,
  data: {
    testDefinitions: [{
      name: "test",
      workflowId: workflowTestUnderTest.id,
      evaluationWorkflowId: null
    }]
  }
})
```